crates/spirv-std/src/vector.rs

```diff
@@ -9,7 +9,7 @@ use glam::{Vec3Swizzles, Vec4Swizzles};
 /// # Safety
 /// Implementing this trait on non-scalar or non-vector types may break assumptions about other
 /// unsafe code, and should not be done.
-pub unsafe trait VectorOrScalar: Default {
+pub unsafe trait VectorOrScalar: Default + Send + Sync + 'static {
```

Comment:
Would it make sense to also throw in `Copy`? (I can't think of a lot of other traits, maybe `PartialEq` but ehhh).

Oh, actually:
https://github.com/Rust-GPU/rust-gpu/blob/8788f924cbc8aa67bd9f9ba0627a6fb5693bc411/crates/spirv-std/src/scalar.rs#L11-L12

So that confirms my expectation that `Copy` should be included. Can you also add `Send + Sync + 'static` to that trait?

Also, why is this trait not actually sealed the same way the `Scalar` trait is? Maybe leftover from the `glam` days? (unless it was sealed and that got undone)

Reply:
Oops I missed the `VectorOrScalar<Scalar = Self>`.

In that case, can you just move `Copy` from `Scalar` to `VectorOrScalar`?

Reply:
I added that trait so likely I just forgot